simplified-app-ekirjasto/src/main/assets/txnative/en/txstrings.json

```diff
@@ -624,6 +900,18 @@
       "contentDescriptionLogo" : {
          "string" : "Logo"
       },
+      "contentDescriptionPage1" : {
+         "string" : "Welcome to the E-library! This service is provided by your local library."
```

Comment:
I don't think this should have "the" in front of "E-library", since it's a proper noun (name)? Compare with "Welcome to the Ikea", that just sounds wrong.

I changed this on the Android side, but should also be fixed in Transifex.

Reply:
Updated the source string, will push the new one into Transifex.